tasks.yaml

```diff
@@ -0,0 +1,89 @@
+metadata:
+  annotations:
+    help: |
+      Install `kit` by following https://github.com/alexec/kit#install.
+      
+      Run `kit up` to start argo.
```

Comment:
Should we include something in the docs about how to run this as well? Looks really cool.

Reply:
You just run `kit up` :)

Reply:
But how do new users of Argo find out about this? Do we expect them to do read the `task.yaml` file?

Reply:
I think we should trial. It is intended to work with Codespaces.